src/Microsoft.ML.Data/Transforms/TypeConverting.cs

```diff
@@ -531,20 +533,21 @@ internal sealed class Defaults
         /// Convinence constructor for simple one column case.
         /// </summary>
         /// <param name="env">Host Environment.</param>
-        /// <param name="inputColumn">Name of the input column.</param>
+        /// <param name="inputColumn">Name of the input column. If set to <see langword="null"/>, the value of the <paramref name="outputColumn"/>
+        /// will be used as input.</param>
         /// <param name="outputColumn">Name of the output column.</param>
         /// <param name="outputKind">The expected type of the converted column.</param>
-        public TypeConvertingEstimator(IHostEnvironment env,
-            string inputColumn, string outputColumn = null,
+        internal TypeConvertingEstimator(IHostEnvironment env,
+            string outputColumn, string inputColumn = null,
```

Comment:
>outputColumn, string inputColumn [](start = 19, length = 32)

Here's another. I'm just going to stop commenting on these, maybe self-review, double check them?